### PR TITLE
Documented GFXReconstruct PR Process

### PR DESCRIPTION
**Pull request by andrew-lunarg**
Adding some info on our process to the CONTRIBUTING.md guide.

Co-authored-by: Andrew Cox <andrew@lunarg.com>

### CODE REVIEW

**Comment by andrew-lunarg** [comment]
I will merge this without further review by end of day if there are no CI failures. Effectively, I am the reviewer of the original source document.

File: CONTRIBUTING.md
Comment (andrew-lunarg) [inline CONTRIBUTING.md:237]: I added this item.

File: CONTRIBUTING.md
Comment (andrew-lunarg) [inline CONTRIBUTING.md:215]: I added the explanation of at least one reason for not cluttering PRs with unrelated changes.

**Comment by andrew-lunarg** [comment]
I needed somewhere to put the images and chose to create a new directory structure: `docs/img` and place then in the leaf of that.

File: CONTRIBUTING.md
Comment (bradgrantham-lunarg) [reply CONTRIBUTING.md:237]: I think this can be omitted.  We won't merge it if we haven't reviewed it enough internally to LunarG.  I think this is our responsibility, not the third-party PR submitter.

**Review by bradgrantham-lunarg — APPROVED** [review]

File: CONTRIBUTING.md
Comment (andrew-lunarg) [reply CONTRIBUTING.md]: Removed

**Comment by andrew-lunarg** [comment]
Should be ready now if it passes CI.

**Pull request merged**
